### DISCUSSION TITLE
为什么cloudflare域名解析出现connect: connection refused，求指点~

### DISCUSSION THREAD

**Discussion by smileljf08** [jeessy2/ddns-go#812] [Q&A]
请求接口https://api.cloudflare.com/client/v4/zones?name=yblhdf.com&status=active&per_page=50失败! ERROR: Get "https://api.cloudflare.com/client/v4/zones?name=yblhdf.com&status=active&per_page=50": dial tcp 127.0.0.1:443: connect: connection refused

![image](https://github.com/jeessy2/ddns-go/assets/26673627/d3bf0278-7045-4cfe-99f2-c80f4c6de396)

**Comment by WaterLemons2k** [comment]
`api.cloudflare.com` 被解析到 `127.0.0.1` 了。

参考：#808 #810

**Reply by smileljf08** [reply]
Nice！真的是这问题，非常感谢~
